scripts/depositor_utils/depositor_bot.py

```diff
@@ -265,12 +270,16 @@ def do_deposit(self):
         except Exception as error:
             logger.error({'msg': f'Deposit failed.', 'error': str(error)})
             DEPOSIT_FAILURE.inc()
+
         else:
             logger.info({'msg': f'Deposited successfully.', 'value': str(result.logs)})
             SUCCESS_DEPOSIT.inc()
 
-        logger.info({'msg': f'Deposit method end. Sleep for 1 minute.'})
-        time.sleep(60)
+        web3.disconnect()
+        web3.provider = HTTPProvider(variables.WEB3_INFURA_PROJECT_ID)
```

Comment:
Does `WEB3_INFURA_PROJECT_ID` contain the full url? It looks like only infra token is passed to `HTTPProvider`